light/provider/http/http_test.go

```diff
@@ -33,23 +33,28 @@ func TestNewProvider(t *testing.T) {
 	require.Equal(t, fmt.Sprintf("%s", c), "http{http://153.200.0.1}")
 }
 
-func NodeSuite(t *testing.T) *node.Node {
+func NodeSuite(t *testing.T) (service.Service, *config.Config) {
```

Comment:
```suggestion
// NodeSuite initiates and runs a full node instance in the background, stopping 
// it once the test is completed
func NodeSuite(t *testing.T) (service.Service, *config.Config) {
```

Reply:
Do you think there would ever be an instance where the test itself wants to pass over the context or is it safe to just always create the context here

Reply:
I think probably we don't want to be creating contexts here and it'd be better to pass it in, but given how we're using contexts at the moment, I think it's safe to increment toward avoiding creating a context in this case.